cosmos/carbon.json

```diff
@@ -71,5 +251,9 @@
     "coinDecimals": 8,
     "coinGeckoId": "switcheo"
   },
-  "features": []
-}
\ No newline at end of file
+  "features": [
```

Comment:
features: any other features that are additionally supported by the chain
cosmwasm: supports CosmWasm smart contracts
secretwasm: supports WASM smart contracts of Secret Network
eth-address-gen: supports EVM account generation
eth-key-sign: supports EVM signatures
axelar-evm-bridge: supports EVM bridge provided by Axelar Network
osmosis-txfees: supports paying fees in other currencies on Osmosis

so Could you remove "ibc-transfer", "ibc-go" field?

Reply:
thanks, corrected